command/v7/unshare_service_command.go

```diff
@@ -0,0 +1,24 @@
+package v7
+
+import "code.cloudfoundry.org/cli/command/flag"
+
+type UnshareServiceCommand struct {
+	BaseCommand
+
+	RequiredArgs    flag.ShareServiceArgs `positional-args:"yes"`
```

Comment:
Not sure if that's a pattern with other commands but, if it is, just ignore this comment.

I see why reusing the arguments from Share makes sense (they are indeed the same) but this has the side effect of, if we ever add more flags to one command, it will inadvertently impacts the other one.